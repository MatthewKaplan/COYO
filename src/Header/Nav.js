import React from 'react';
import '../Styles/Nav.scss';
import logo from './logo.png'
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'


export default function Nav(props) {
  return (
    <nav className="navBar">
      <div className="logo">
        <a href="#home" className="logoLink"> <img src={logo} alt="CoYo Logo" className="logoImg"/></a>
      </div>
      <ul className="navList">
        <li className="facebook socialMedia">
         <a href="https://www.facebook.com">
          <FontAwesomeIcon icon={['fab', 'facebook-square']} />
         </a>
        </li>
        <li className="twitter socialMedia">
         <a href="www.twitter.com">
          <FontAwesomeIcon icon={['fab', 'twitter']} />
         </a>
        </li>
        <li className="instagram socialMedia">
         <a href="www.instagram.com">
          <FontAwesomeIcon icon={['fab', 'instagram']} />
         </a>
        </li>
      </ul>
    </nav>
  )
}