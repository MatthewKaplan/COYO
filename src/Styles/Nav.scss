@import 'Mixins';

ul,
li {
   list-style-type: none;
   margin: 0;
   padding: 0;
}

.navBar {
  // height: 50px;
  background: #051624 ;
  padding: 0 5px;
  display: flex;
  justify-content: space-between;
  align-items: center;
  color: #EFF0F2;
}

.navList{
  display: flex;
  width: 10%;
  justify-content: space-around;
  font-size: 30px;
}


.logo {
  width: 3rem;
  border: 1px solid  #EFF0F2;
  // border-radius: 5px;
  margin: 10px;
}

.logoImg {
  width: 100%;
}