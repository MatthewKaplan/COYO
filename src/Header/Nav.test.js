import React from 'react';
import { Nav } from './Nav';
import { shallow } from 'enzyme';

describe('Nav', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<Nav data={{}} />);
    expect(wrapper).toMatchSnapshot();
  });
});