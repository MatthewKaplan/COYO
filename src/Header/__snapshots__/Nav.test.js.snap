// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav should match snapshot 1`] = `
<nav
  className="navBar"
>
  <div
    className="logo"
  >
    <a
      className="logoLink"
      href="#home"
    >
       
      <img
        alt="CoYo Logo"
        className="logoImg"
        src="logo.png"
      />
    </a>
  </div>
  <ul
    className="navList"
  >
    <li
      className="facebook socialMedia"
    >
      <a
        className="icon"
        href="https://www.facebook.com"
        title="link to facebook"
      >
        <FontAwesomeIcon
          border={false}
          className=""
          fixedWidth={false}
          flip={null}
          icon={
            Array [
              "fab",
              "facebook-square",
            ]
          }
          inverse={false}
          listItem={false}
          mask={null}
          pull={null}
          pulse={false}
          rotation={null}
          size={null}
          spin={false}
          symbol={false}
          title=""
          transform={null}
        />
      </a>
    </li>
    <li
      className="twitter socialMedia"
    >
      <a
        className="icon"
        href="www.twitter.com"
        title="link to Twitter"
      >
        <FontAwesomeIcon
          border={false}
          className=""
          fixedWidth={false}
          flip={null}
          icon={
            Array [
              "fab",
              "twitter",
            ]
          }
          inverse={false}
          listItem={false}
          mask={null}
          pull={null}
          pulse={false}
          rotation={null}
          size={null}
          spin={false}
          symbol={false}
          title=""
          transform={null}
        />
      </a>
    </li>
    <li
      className="instagram socialMedia"
    >
      <a
        className="icon"
        href="www.instagram.com"
        title="link to Instagram"
      >
        <FontAwesomeIcon
          border={false}
          className=""
          fixedWidth={false}
          flip={null}
          icon={
            Array [
              "fab",
              "instagram",
            ]
          }
          inverse={false}
          listItem={false}
          mask={null}
          pull={null}
          pulse={false}
          rotation={null}
          size={null}
          spin={false}
          symbol={false}
          title=""
          transform={null}
        />
      </a>
    </li>
  </ul>
</nav>
`;
